doc/developer-documentation.md

```diff
@@ -170,6 +170,35 @@ Here, `node` refers to the name of API PubSub channel where node events are publ
 
 After following these steps, run your pipeline instance to activate your newly added test configuration.
 
+### Example to enable a new kselftest
+
+The kselftests (found in `<linux_kernel>/tools/testing/selftests`) are enabled suite by suite. Most parameters are already set for other kselftests. Those must be reused while enabling new kselftests which make enabling new tests pretty straight forward. The `dt` kselftest suite can be enabled by:
+
+1. Create a job definition in `jobs` section.
+```
+  kselftest-dt:
+    template: kselftest.jinja2
+    kind: job
+
+    params:
+      nfsroot: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240221.0/{debarch}'
+      collections: dt
+      job_timeout: 10
+    rules:
+      min_version:
+        version: 6
+        patchlevel: 7
+    kcidb_test_suite: kselftest.dt
+```
+2. Create a `scheduler` entry for `kselftest-dt` job. Lets schedule this suite in `lava-collabora` lab at `kbuild-gcc-12-arm64` build completion event. The entry by using already defined anchors would be:
```

Comment:
```suggestion
2. Create a `scheduler` entry for `kselftest-dt` job. Let's schedule this suite in `lava-collabora` lab at `kbuild-gcc-12-arm64` build completion event. The entry by using already defined anchors would be:
```

Reply:
This isn't resolved yet.

Reply:
Seems like there was issue with force push. Updated now.